UI/src/controller/trafficSignsCollection.js

```diff
@@ -34,16 +34,10 @@
         additionalPanels: { values : [45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 147, 146, 145, 144, 143, 142, 141, 140, 139, 138, 148, 149, 150, 151], groupName: 'Lisakilvet'}
     };
 
-    var trafficSignsTurnRestriction = [10, 11, 12];
```

Comment:
Is this remove ok ?

Reply:
This array here just contains the information of trafficSignsTurnRestriction but there is no individual use of this. It is simply the same as trafficSignRestriction and the idea of use is exactly the same. So I decided to have both on one array/function.

Reply:
After talking with team this is now more clearer.
Nothing changed here